constructor/main.py

```diff
@@ -177,6 +180,27 @@ def main_build(
                     new_extras.append({orig: dest})
         info[extra_type] = new_extras
 
+    def has_frozen_file(extra_files: list[str | dict[str, str]]) -> bool:
+        def check_path(path_str: str) -> bool:
+            path = Path(path_str)
+            return path.name == "frozen" and path.parent.name == "conda-meta"
+
+        for file in extra_files:
+            if isinstance(file, str) and check_path(file):
+                return True
+            elif isinstance(file, dict) and any(check_path(val) for val in file.values()):
+                return True
+        return False
+
+    if (
+        has_frozen_file(info.get("extra_files", []))
+        and exe_type == StandaloneExe.CONDA
+        and check_version(exe_version, min_version="25.5.0", max_version="25.7.0")
```

Comment:
```suggestion
        and not check_version(exe_version, min_version="25.5.0")
```

I think?

Reply:
I originally went this route but was told to only flag 25.5.x and not flag any older versions, though this makes the most sense to me unless there is something else I'm missing. CC: @marcoesters for clarity

Reply:
Ok, sorry I mixed the two. We can close this thread then.